Listener/LicenseListener.php

```diff
@@ -57,16 +58,19 @@ public function onKernelRequest(GetResponseEvent $event)
             // Checking for whitelisted users
             try {
                 $user = $this->tokenStorage->getToken()->getUser();
-                $today = date('Y-m-d');
-                if ($user instanceof UserInterface) {
+                if ($user instanceof Tenant) {
+                    if($user->isWhiteListed()){
+                        return;
+                    }
+
+                    $today = date('Y-m-d');
                     $whitelist = $this->kernel->getContainer()->getParameter('license_whitelist');
                     foreach ($whitelist as $allowed) {
```

Comment:
well, current implementation allow to use "both ways" - database and parameters.yml
Leaving code for processing parameters.yml will give full backward compatibility.
So, basically, you could simply update lib and not required to change  parameters.yml
Maybe lets leave that code for BC?